### ISSUE TITLE
[Screen Reader - Data Explorer (iframe)] Screen reader does not announcing ‘’Command lines’ information present in ‘editor pane’ under new documents.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1777]
**URL**:https://ms.portal.azure.com/

**User Experience:**
User who depend on screen reader might get confused and will not able to get proper information about the controls if screen reader does not announce command line information.

**Pre-Requisite:**
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

**Repro-Steps**
1. Open URL: https://ms.portal.azure.com/
2. Login via valid credentials
3. Navigate to "Azure Cosmos DB " from left pane and select it.
4. Navigate to any of the Azure Cosmos DB which is already created and select it.
5. Navigate to "Data Explorer" from the left and select it.
6. Navigate through all the controls under SQL API Section and select Documents item.
7. Navigate to "New Document" button and select it.
8. Editor Pane will open.
9. Enable screen reader and verify whether screen reader announcing commend line or not. 

**Actual Result:**
Screen reader does not announcing ‘’Command lines’ information present in ‘editor pane’ under new documents

**Expected Result:**
Screen reader should announce the ‘’Command lines’ information as 'Command line 1 …etc.' present in ‘editor pane’ under new documents

**Note:**

1.Similar issue exist with chrome + JAWS, Chrome + NVDA.
2.Similar issue exist through out the application.

**MAS Reference:**
MAS 1.3.1 - Info and Relationships (40)

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug.

Bug copied from Data explorer iframe : https://msdata.visualstudio.com/CosmosDB/_workitems/edit/525374

For more information please use this Link.
![MAS1 3 1_Command line](https://user-images.githubusercontent.com/30456910/73053797-bc251580-3eae-11ea-8bc0-33c6ddc002df.PNG)

**Comment by alexdima** [comment]
Duplicate of #410
